third_party/abigen-celo/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/certusone/wormhole/third_party/abigen-celo
+
+go 1.15
```

Comment:
That's a pretty old version of Go - how comes?

Reply:
This was copied from the existing abigen version of this file. It is only used to build the celo abigen tool, which works fine with this version. If / when we need to regenerate the ABIs for both the go-ethereum and celo-ethereum watchers, we can upgrade both. Leaving this as is for now.